timeserio/keras/multinetwork.py

```diff
@@ -71,11 +71,17 @@ def _callbacks(self, **kwargs) -> Dict[str, "keras.callbacks.Callback"]:
     @property
     def _funcs_with_legal_params(self):
         Sequential = keras.models.Sequential
-        return [
+        methods = [
             Sequential.fit, Sequential.fit_generator, Sequential.predict,
-            Sequential.predict_classes, Sequential.evaluate, self._model,
-            self._callbacks
+            Sequential.predict_classes, Sequential.evaluate
         ]
+        # fix for tensorflow 2.2.0 using method wrappers
+        # these were removed again in 2.3.0
+        unwrapped_methods = [
+            getattr(method, "__wrapped__", method)
```

Comment:
can you do `getattr(method, "__wrapped__", default=method)`? just to be super clear that it falls back on `method` in the absence of `__wrapped__`